src/renderer/components/Select.vue

```diff
@@ -54,6 +54,10 @@ export default {
     },
 
     toggleOpen() {
+      if (this.isDisabled) {
```

Comment:
Instead of this, you could make the toggle element a button and disable it like so:

```
<button :disabled="isDisabled" />
```

Reply:
Can you explain what is wrong with this approach?

Reply:
Willing to change but would like to understand the reason.

Reply:
Note that changing to a button will likely style it differently when it is disabled, (and as a button and not a <ul>)

Reply:
Its not a show-stopper, just a suggestion. Your call, ultimately.

Reply:
Switched to button in latest commit